test/exhal/collection_test.exs

```diff
@@ -84,6 +86,31 @@ defmodule ExHal.CollectionTest do
     assert ExHal.to_stream(doc) |> is_a_stream
   end
 
+  describe "Non Hal Responses" do
```

Comment:
Had to wrap this in a describe block so the setup doesn't affect other tests. Open to a different way to do this.